src/generators/genjvm.ml

```diff
@@ -73,6 +73,7 @@ type generation_context = {
 	anon_identification : jsignature tanon_identification;
 	preprocessor : jsignature preprocessor;
 	default_export_config : export_config;
+	nadako : jsignature nadako_interfaces;
```

Comment:
I don't want to not give credit to nadako, but let's have a proper name for this …

Reply:
this is how asteroids are named after the discoverer